mpvplayer/player.go

```diff
@@ -376,8 +376,8 @@ func (p *Player) IsSeeking() (bool, error) {
 	return false, nil
 }
 
-func (p *Player) SeekAbsolute(float64) error {
-	return nil
+func (p *Player) SeekAbsolute(position int) error {
+	return p.instance.Command([]string{"seek", strconv.Itoa(position), "absolute"})
```

Comment:
This isn't used, and the change is not required for this patch. It was another attempt to get seek working, but it also failed. However, this _is_ how `SeekAbsolute` _should_ work, according to the Lua implementation in the mpv project. Hopefully, it goes away when we switch to the external OpenSubsonic library, but in the meantime, maybe it'll be useful, and it does flesh out the bespoke ABI.